sat/cli/bmccreds/parser.py

```diff
@@ -102,6 +122,29 @@ def add_bmccreds_subparser(subparsers):
              '--password.'
     )
 
+    bmccreds_subparser.add_argument(
+        '--password-length',
+        help='The desired length of the random BMC password. Only valid with '
+             '--random-password.'
```

Comment:
Let's add a check to `validate_args` that checks whether `--random-password` is specified with these.